src/react-webcam.js

```diff
@@ -298,7 +309,9 @@ export default class Webcam extends Component {
         muted={this.props.audio}
         className={this.props.className}
         playsInline
-        style={this.props.style}
+        style={Object.assign({}, this.props.style, this.props.rotate ? {
```

Comment:
The only problem that I can see with this currently, is that any existing transform property would be overwritten by the `rotate(${this.props.rotate}deg)`. Ideally, the rotate would be appended if a transform property already existed.